### ISSUE TITLE
How to set Tab Index for keyboard navigation?

### ISSUE THREAD

**Issue by dlynnsmith** [trendmicro-frontend/react-sidenav#50]
I am doing an accessibility review of our new website.  I have a major flaw at the moment: The main navigation from the SideNav cannot be reached via the keyboard (tab to the menu item, press Enter to fire the link).  What am I doing wrong?

Here is an example NavItem with the tab index trying to be defined:
`                  <NavItem tabindex="5" eventKey="myutilities">
                    <NavIcon >
                      <Industry32 className={"my-custom-class"}/>
                    </NavIcon>
                    <NavText>
                      My Utilities
                    </NavText>
                  </NavItem>
`
The resulting page source has tabindex="-1", no matter what index number I put on an item.  I've attached a screen shot of the Inspect Element results.
![Screen Shot 2020-02-27 at 2 56 35 PM](https://user-images.githubusercontent.com/16374369/75481656-66420280-5971-11ea-8f45-078ee42e76b4.png)

TabIndex *shows* better when put on the NavIcon or the NavText.  But those don't actually fire the link when the Enter key is pressed.

I'm on a tight deadline, and this is a major accessibility issue for our app.  I hope there is a simple fix.  Will you enlighten me, please?  :-)

Thank you!

**Comment by roth1002** [comment]
We hard code tabIndex to -1
Need to refine this attribute

https://github.com/trendmicro-frontend/react-sidenav/blob/e21ab4afc63c773b357cc9247f958b534838c43c/src/NavItem.jsx#L150

**Comment by dlynnsmith** [comment]
When might this bug be fixed?  Our new app must go to beta testing next month, and to full public production by July.  Without being able to tab thru the sidenav (and so requiring mouse clicks for all our major navigation), we don't meet accessibility standards.

Can you suggest a workaround if the bug isn't fixed?

**Comment by dlynnsmith** [comment]
Our launch was delayed due to COVID-19 but is getting started back.  This issue still needs to be corrected.  Has any progress been made on your end?

How are other sites dealing with accessibility while using this sidenav?
